vinyldns/resource_record_set.go

```diff
@@ -33,6 +33,7 @@ func resourceVinylDNSRecordSet() *schema.Resource {
 			"zone_id": &schema.Schema{
 				Type:     schema.TypeString,
 				Required: true,
+				ForceNew: true,
```

Comment:
I am not seeing this used in this PR?

Reply:
@pauljamescleary Did you notice the docs I linked to in the PR message?

This explains & my tests seek to ensure that it does in fact function as we intend:
https://www.terraform.io/docs/extend/schemas/schema-behaviors.html#forcenew